src/it/ajc/verify.groovy

```diff
@@ -30,3 +30,5 @@
 
 assert !new File(basedir, 'target/classes/jcabi-ajc.log').exists()
 assert new File(basedir, 'target/jcabi-ajc.log').exists()
+assert new File(basedir, 'target/classes/com/jcabi/foo/Sample.class').exists()
+assert new File(basedir, 'target/unwoven/Sample.class').exists()
```

Comment:
@krzyk When copying unwoven classes to a sepparate directory, do we need to preserve the package folders or is it enough if we copy the .class files only?